RELEASES.md

```diff
@@ -45,7 +45,7 @@ the 0.38.x line.
 1. Start on `main`
 
 2. Ensure that there is a [branch protection
-   rule](https://github.com/tendermint/tendermint/settings/branches) for the
+   rule](https://github.com/tendermint/tendermint/branches) for the
```

Comment:
Technically the original link was correct, but it may be showing up as a 404 because you need admin privileges to access it. Let's rather substitute with GitHub's documentation here.

```suggestion
   rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule) for the
```